aws/table_aws_bedrock_imported_model.go

```diff
@@ -0,0 +1,235 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/bedrock"
+	"github.com/aws/aws-sdk-go-v2/service/bedrock/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsBedrockImportedModel(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_bedrock_imported_model",
+		Description: "AWS Bedrock Imported Model",
+		List: &plugin.ListConfig{
+			Hydrate: listBedrockImportedModels,
+			Tags:    map[string]string{"service": "bedrock", "action": "ListImportedModels"},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "creation_time", Require: plugin.Optional, Operators: []string{">", ">=", "<", "<="}},
+			},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "arn", Require: plugin.AnyOf},
+				{Name: "model_name", Require: plugin.AnyOf},
+			},
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
+			},
+			Hydrate: getBedrockImportedModel,
+			Tags:    map[string]string{"service": "bedrock", "action": "GetImportedModel"},
+		},
```

Comment:
Could you please add the `HydrateConfig` for the `getBedrockImportedModel` function, since it’s being used as a hydrate function for several columns?